users/serializers.py

```diff
@@ -92,6 +92,24 @@ def create(self, validated_data: Any) -> Any:
         send_email("verify_email.html", email_data)
         return user
 
+    def to_representation(self, instance: Any) -> Any:
+
+        """check if the user is being followed and return the following field as true or false"""
+
+        if self.context.get("request").user.is_authenticated:  # type: ignore[union-attr]
```

Comment:
store request in a variable cause you are using it multiple times instead of using self.context.... everywhere

Reply:
done